Common/Extensions.cs

```diff
@@ -2141,6 +2141,8 @@ public static BaseData Scale(this BaseData data, Func<decimal, decimal> factor)
                         if (securityType == SecurityType.Equity)
                         {
                             tick.Value = factor(tick.Value);
+                            tick.BidPrice = factor(tick.BidPrice);
+                            tick.AskPrice = factor(tick.AskPrice);
                         }
                         if (securityType == SecurityType.Option
```

Comment:
I think we could directly add `securityType == SecurityType.Equity` in the if bellow?